libminifi/test/persistence-tests/PersistenceTests.cpp

```diff
@@ -257,8 +258,26 @@ TEST_CASE("Persisted flowFiles are updated on modification", "[TestP1]") {
   config->set(minifi::Configure::nifi_flowfile_repository_directory_default, utils::file::FileUtils::concat_path(dir, "flowfile_repository"));
 
   std::shared_ptr<core::Repository> prov_repo = std::make_shared<TestRepository>();
-  std::shared_ptr<core::repository::FlowFileRepository> ff_repository = std::make_shared<core::repository::FlowFileRepository>("flowFileRepository");
-  std::shared_ptr<core::ContentRepository> content_repo = std::make_shared<core::repository::FileSystemRepository>();
+  std::shared_ptr<core::Repository> ff_repository;
+  std::shared_ptr<core::ContentRepository> content_repo;
+  SECTION("FlowFileRepository"){
+    ff_repository = std::make_shared<core::repository::FlowFileRepository>("flowFileRepository");
+    SECTION("VolatileContentRepository"){
+      content_repo = std::make_shared<core::repository::VolatileContentRepository>();
+    }
+    SECTION("FileSystemContenRepository"){
+      content_repo = std::make_shared<core::repository::FileSystemRepository>();
+    }
+  }
+  SECTION("VolatileFlowFileRepository"){
+    ff_repository = std::make_shared<core::repository::FlowFileRepository>("volatileFlowFileRepository");
```

Comment:
should this be an instance of `VolatileFlowFileRepository`?

Reply:
indeed it should